docs/pages/services/indexer.mdx

```diff
@@ -192,22 +325,51 @@ Meaning that query 0 is for everything in the `World` at address `0x6e9474e9c836
 
 </details>
 
-## Resetting the database
+## Some examples
 
-When you restart the blockchain you _have_ to reset the database, otherwise you'll get a mix of old and new information.
-Here are the directions how to do it.
+### SQLite to view the tasks from the React template
 
-### SQLite
+- Start the indexer.
 
-Delete the database file (by default `anvil.db`).
+  ```sh copy
+  pnpm install @latticexyz/store-indexer@next
+  export RPC_HTTP_URL=http://127.0.0.1:8545
+  pnpm sqlite-indexer
+  ```
 
-### PostgreSQL
+- Read the data.
 
-Run `psql` and run this command:
+  ```sh copy
+  curl 'http://localhost:3001/trpc/findAll?batch=1&input=%7B%220%22%3A%7B%22json%22%3A%7B%22chainId%22%3A31337%2C%22address%22%3A%220x6e9474e9c83676b9a71133ff96db43e7aa0a4342%22%7D%7D%7D' | \
+  jq '.[0].result.data.json.tables' | \
+  jq -c '.[] | select (.name=="Tasks")' | \
+  jq .records | more
+  ```
 
-```sql
-DROP OWNED BY <see below>;
-```
+### Read data from SkyStrife season 0
```

Comment:
What i had in mind with the sky strife data was to run the indexer on a world with more data to showcase the indexer